.storybook/main.ts

```diff
@@ -4,7 +4,11 @@ const path = require("path");
 const fs = require("fs");
 const glob = require("fast-glob");
 
-module.exports = {
+import type {StorybookConfig} from "@storybook/react-webpack5";
+
+const config: StorybookConfig = {
+    framework: "@storybook/react-webpack5",
```

Comment:
This was at the bottom of the file, but to me, this feels like the most important thing so I've moved it to the top!